build-bench-env.sh

```diff
@@ -264,6 +293,15 @@ if test "$setup_iso" = "1"; then
   popd
 fi
 
+if test "$setup_scudo" = "1"; then
+  partial_checkout scudo $version_scudo scudo https://github.com/llvm/llvm-project "compiler-rt/lib/scudo/standalone"
+  cd "compiler-rt/lib/scudo/standalone"
+  # TODO: make the next line prettier instead of hardcoding everything.
+  clang++ -fPIC -std=c++14 -fno-exceptions -fno-rtti -I include -shared -o libscudo.so *.cpp -pthread
```

Comment:
So the original line in @joshtriplett's comment:
```
clang++ -flto -fuse-ld=lld -fPIC -std=c++14 -fno-exceptions -fno-rtti -fvisibility=internal -msse4.2 -O3 -I include -shared -o libscudo.so *.cpp -pthread
```
Includes some settings you are missing.  The `-O3` is particularly important. Probably also the `-flto` related ones too.  Building using your current setting seems much slower. For instance, cfrac with the original settings was:
```
cfrac scudo 08.41 4628 8.40 0.00 0 604
```
with your current settings
```
cfrac scudo 45.45 4692 45.42 0.00 6 606
```